Add docs for delegatedPermissionClassifications and permissionGrantPolicies (#10183)
microsoftgraph/microsoft-graph-docs: api-reference/beta/api/permissiongrantpolicy-delete-excludes.md

---
title: "Delete an excludes condition set from a permission grant policy"
description: "Deletes an excluded condition set from permission grant policy."
localization_priority: Normal
doc_type: apiPageType
ms.prod: "microsoft-identity-platform"
author: "psignoret"
---

> # Delete an excludes condition set from a permission grant policy

Reviewer: We use resource names in H1 titles, so this should be "Delete permissionGrantConditionSet".

---

> title: "Delete an excludes condition set from a permissionGrantPolicy"

Reviewer 2: ```suggestion
title: "Delete an excludes condition set from a permission grant policy"
```

description: "Deletes an excluded condition set from permission grant policy."
localization_priority: Normal
doc_type: apiPageType
ms.prod: "microsoft-identity-platform"
author: "psignoret"
---

> # Remove a excludes condition set from a permission grant policy

Reviewer 2: ```suggestion
# Delete an excludes condition set from a permission grant policy
```

Namespace: microsoft.graph

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]

Deletes a [permissionGrantConditionSet](../resources/permissiongrantconditionset.md) from the **excludes** collection of a [permissionGrantPolicy](../resources/permissiongrantpolicy.md).

## Permissions

One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](/graph/permissions-reference).

| Permission type      | Permissions (from least to most privileged)              |
|:--------------------|:---------------------------------------------------------|
| Delegated (work or school account) | PermissionGrantPolicy.ReadWrite.All |
| Delegated (personal Microsoft account) | Not supported.    |
| Application | PermissionGrantPolicy.ReadWrite.All |

## HTTP request

<!-- { "blockType": "ignored" } -->

```http

> DELETE /policies/permissionGrantPolicies/{id}/excludes/{id}

Reviewer 2: ```suggestion
DELETE /policies/permissionGrantPolicies/{permissiongrantpolicy-id}/excludes/{exclude-id}
```

Author: The *id* property of a permissionGrantPolicy is not a Guid. There is no Guid identifier for a permissionGrantPolicy, so there is no ambiguity.

```

## Request headers

| Name       | Type | Description|
|:---------------|:--------|:----------|
| Authorization  | string  | Bearer {token}. Required. |

## Request body

Do not supply a request body for this method.

## Response

> If successful, this method returns `204 No Content` response code. It does not return anything in the response body.

Reviewer 2: ```suggestion
If successful, this method returns a `204 No Content` response code. It does not return anything in the response body.
```

## Examples

### Request

> Here is an example of the request to delete an **excludes** condition set from a custom permission grant policy.

Reviewer 2: ```suggestion
The following is an example of the request.
```

> # [HTTP](#tab/http)

Reviewer 2: Please remove line 56. This header is automatically added when the SDK code snippets are added.

<!-- {
  "blockType": "request",
  "name": "permissiongrantpolicy_delete_excludes"
}-->

```http

> DELETE https://graph.microsoft.com/beta/policies/permissionGrantPolicies/my-custom-consent-policy/excludes/6a846635-3e70-4a10-821e-512a0db93cbd

Reviewer 2: Id's are typically in GUID form. Should "my-custom-consent-policy" be a GUID?

Author: No, **id** of permission grant policies are actually strings like in the examples.

```

> ---

Reviewer 2: This line is also a copy paste issue. Line 67 should be removed and will be added back by the auto-generation of code snippets.
